### DISCUSSION TITLE
php 8 update

### DISCUSSION THREAD

**Discussion by darkjezza** [AscensionGameDev/Intersect-Engine#1822] [Q&A] [answered]
any chance to update the php version of update.php and stream.php to php 8?

**Comment by pandinocoder** [comment, marked as answer]
The PHP script is not a part of the official repo, this isn't the right place to ask about it.

Someone updated it for PHP 8.1 [on Discord](https://discord.com/channels/363106200243535872/708744199771783219/1128381611600257174).
